app/background-process/ui/downloads.js

```diff
@@ -61,7 +61,9 @@ export function registerListener (win, opts = {}) {
     }
 
     downloads.push(item)
-    downloadsEvents.emit('new-download', toJSON(item))
+
+    if(!item.name.includes(".html"))
```

Comment:
When the `'will-download'` event is emitted it triggers the `browser-dropdown-menu` to open. The one in the top right of the shell. With  the Library, Bookmarks, etc options. 

I felt like this shouldn't happen when downloading an html page as it will download for most users in milliseconds.